firmware/quintuna/VC/src/app/states/app_inverterOnState.c

```diff
@@ -31,12 +33,27 @@ static void inverterOnStateRunOnEntry(void)
 static void inverterOnStateRunOnTick100Hz(void)
 {
     // here we need to check if the inverters are alive and are sending us active can messages
-
     const bool inverters_bsystemReady = app_canRx_INVFL_bSystemReady_get() && app_canRx_INVFR_bSystemReady_get() &&
                                         app_canRx_INVRL_bSystemReady_get() && app_canRx_INVRR_bSystemReady_get();
+
     if (inverters_bsystemReady)
-    { // hwere we also need to check if the bus voltage is also above the dc cap voltage
-        app_stateMachine_setNextState(&bmsOn_state);
+    { // here we also need to check if the bus voltage is also above the dc cap voltage
+        if (app_canTx_VC_Info_InverterRetry_get())
+        {
+            // TODO do we want to go back to hvInit straight or go through PCM to do a quick check to see if everything
+            // is ok
+            app_stateMachine_setNextState(&pcmOn_state);
+        }
+        else
+        {
+            app_stateMachine_setNextState(&bmsOn_state);
+        }
+    }
+    // If we are retrying the inverters and we are coming from drive just send the messages to unset the inverter faults
+    // instead of turning them off
+    else if (app_canTx_VC_Info_InverterRetry_get())
+    {
+        app_warningHandling_inverterRest();
     }
```

Comment:
is this comment true rn? inverterReset does this:
```
        inverter_reset_handle[inverter].can_invOn(false);
        inverter_reset_handle[inverter].can_dcOn(false);
        inverter_reset_handle[inverter].can_enable_inv(false);
```
which looks like you're turning it off? or does this just delatch faults?